hack/lib/init.sh

```diff
@@ -30,18 +31,21 @@ servicecertsigner/v1alpha1 \
 template/v1 \
 user/v1 \
 webconsole/v1 \
+vendor/k8s.io/kubelet/config/v1beta1 \
 "
 API_PACKAGES="\
 github.com/openshift/api/apps/v1,\
 github.com/openshift/api/authorization/v1,\
 github.com/openshift/api/build/v1,\
 github.com/openshift/api/image/v1,\
+github.com/openshift/api/machineconfiguration/v1,\
 github.com/openshift/api/network/v1,\
 github.com/openshift/api/oauth/v1,\
 github.com/openshift/api/project/v1,\
 github.com/openshift/api/quota/v1,\
 github.com/openshift/api/route/v1,\
 github.com/openshift/api/security/v1,\
 github.com/openshift/api/template/v1,\
-github.com/openshift/api/user/v1\
+github.com/openshift/api/user/v1,\
+k8s.io/kubelet/config/v1beta1\
```

Comment:
this is to run protobuf on the vendored kubelet types.

Reply:
This is not good. Make a patch against origin and let the publishing bot publish back to our kubelet fork. Then revendor here.

Reply:
Why do we need protobuf for machine configs? Those are CRDs, which do not support protobuf anyway.

Reply:
I just remove this and don't need to do any that now, is that right?

Reply:
@sttts can you elaborate a little here? I need the kubelet config in this list, otherwise I'm running into this:
```
go build github.com/openshift/api/...
# github.com/openshift/api/machineconfiguration/v1
machineconfiguration/v1/generated.pb.go:194:10: undefined: KubeletAnonymousAuthentication
machineconfiguration/v1/generated.pb.go:195:8: undefined: KubeletAnonymousAuthentication
machineconfiguration/v1/generated.pb.go:196:8: undefined: KubeletAnonymousAuthentication
machineconfiguration/v1/generated.pb.go:200:10: undefined: KubeletAuthentication
machineconfiguration/v1/generated.pb.go:201:8: undefined: KubeletAuthentication
machineconfiguration/v1/generated.pb.go:202:8: undefined: KubeletAuthentication
machineconfiguration/v1/generated.pb.go:204:10: undefined: KubeletAuthorization
machineconfiguration/v1/generated.pb.go:205:8: undefined: KubeletAuthorization
machineconfiguration/v1/generated.pb.go:206:8: undefined: KubeletAuthorization
machineconfiguration/v1/generated.pb.go:208:10: undefined: KubeletConfiguration
machineconfiguration/v1/generated.pb.go:208:10: too many errors
```
What do I need to do in the origin repo?